src/main/java/com/google/sps/data/PollingStation.java

```diff
@@ -60,11 +66,11 @@ public static Builder builder() {
 
     public abstract Builder setLocationType(String type);
 
+    public abstract Builder setSources(ImmutableList<String> sources);
```

Comment:
Nit: I think you can just pass in the list, and the builder will do the copy to an immutablelist internally.